xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/service/BitfinexAdapters.java

```diff
@@ -103,7 +103,14 @@ private BitfinexAdapters() {}
   }
 
   public static String adaptBitfinexCurrency(String bitfinexSymbol) {
-    return bitfinexSymbol.toUpperCase();
+    String currency = bitfinexSymbol.toUpperCase();
+    if (currency.equals("DSH")) {
+      currency = "DASH";
+    }
+    if (currency.equals("QTM")) {
+      currency = "QTUM";
+    }
+    return currency;
```

Comment:
don't, until we have a global solution for unifying currency code exchanges modules should use code actually used by the implemented exchange